README.md

```diff
@@ -1,16 +1,36 @@
 # React Design Patterns
 
-This repository holds the source code for some [PX Blue's design patterns](https://pxblue.github.io/patterns), written in React. 
+This repository holds the source code for some [PX Blue's design patterns](https://pxblue.github.io/patterns), written in React. The code based is arranged so that you can easily copy and paste the code examples into your own projects.
 
-> NOTE: once this goes public, we will be able to provide deeplinks into each pattern using CodeSandbox.
+## Current Patterns
+
+| Pattern                                                                        | View on Code Sandbox                                                                                                                     |
+| ------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| [Action List](https://pxblue.github.io/patterns/lists)                         | [Link](https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/lists/action-list/index.tsx)             |
```

Comment:
I believe you can modify these deep links to also specify which file to land on in the editor and which page to open in the preview...might be nice to open the root file for each pattern instead of App.tsx and show the example page that we're linking to.

I think they use `&module=` for the file to open and `&initialPath=` to set the route to land on.